src/components/Display.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display({ result }) {
+  return (
+    <div>
+      <h2>{result}</h2>
+    </div>
+  )
+};
+
+Display.prototype = {
+  name: PropTypes.String
+};
+
```

Comment:
- [ ] This props type declaration has typos, we need to change to this one instead:
```suggestion
Display.propTypes = {
  result: PropTypes.string,
};

```

Reply:
Thank you for the detailed feedback, really appreciate it 👍